cassandra/client.go

```diff
@@ -277,21 +287,45 @@ func convert(i interface{}) (interface{}, error) {
 	return num, err
 }
 
-func createCluster(server string) *gocql.ClusterConfig {
-	cluster := gocql.NewCluster(server)
+func createCluster(config clientOptions) *gocql.ClusterConfig {
+	cluster := gocql.NewCluster(config.server)
 	cluster.Consistency = gocql.One
 	cluster.ProtoVersion = 4
+
+	cluster.Timeout = config.timeout
+	cluster.ConnectTimeout = config.connectionTimeout
+
+	cluster.DisableInitialHostLookup = !config.initialHostLookup
+	cluster.IgnorePeerAddr = config.ignorePeerAddr
+
+	if config.ssl != nil {
+		cluster = addSslOptions(cluster, config.ssl)
+	}
+
 	return cluster
 }
 
-func getSession(co clientOptions) *gocql.Session {
-	cluster := createCluster(co.server)
+func createKeyspace(co clientOptions) {
+	cluster := createCluster(co)
+	session, err := cluster.CreateSession()
```

Comment:
I know, but you need to create keyspace separately. Your configured session isn't able to switch to new keyspace. This case will remove keyspace name from rest of queries. Another approach is to format each query with keyspace name. What do you think?